bin/needs-a-genny

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+genny_path = ->(slug) {
+  ['exercises', slug, '.meta', 'generator'].join('/')
+}
+
+genny_path_exists = ->(slug) { !Dir.exist?(genny_path[slug]) }
+
+puts Dir.children('exercises')
+       .filter(&genny_path_exists).sort.join("\n")
```

Comment:
Indentation here, but otherwise good to go.

Reply:
I wasn't sure if it should indent relative to `Dir` or `puts`. Thanks for the clarification.

Reply:
Since technically this line is evaluated as:

`puts(Dir.children('exercises').filter().sort.join())`  The dots should be aligned together, it is inclusive of the argument given, not at the level of puts.

```suggestion
        .filter(&genny_path_exists).sort
```